server/src/main/java/org/elasticsearch/cluster/metadata/MappingMetadata.java

```diff
@@ -18,11 +18,13 @@
 import org.elasticsearch.common.xcontent.XContentHelper;
 import org.elasticsearch.index.mapper.DocumentMapper;
 import org.elasticsearch.index.mapper.MapperService;
+import org.elasticsearch.index.mapper.MappingLookup;
```

Comment:
Do we still need these changes? Things are now tracked in IndexMetadata.

Reply:
We don't, thanks for the catch! 💯